runtime/vimrc_example.vim

```diff
@@ -1,21 +1,17 @@
 " An example for a vimrc file.
 "
 " Maintainer:   Bram Moolenaar <Bram@vim.org>
-" Last change:  2014 Nov 05
+" Last change:  2015 Jun 29
 "
```

Comment:
Bram isn't the maintainer of this anymore, so I think lines 2-4 should be deleted.